lib/FileUrl.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+const { Check, DeveloperError } = require('cesium');
+const os = require('os');
+const path = require('path');
+const { domainToUnicode } = require('url');
+
+module.exports = {
+    fileURLToPath: fileURLToPath,
+    pathToFileURL: pathToFileURL
+};
+
+const isWindows = os.platform() === 'win32';
+const forwardSlashRegEx = /\//g;
+
+const CHAR_LOWERCASE_A = 97;
+const CHAR_LOWERCASE_Z = 122;
+const CHAR_FORWARD_SLASH = 47;
+const CHAR_BACKWARD_SLASH = 92;
+
+const percentRegEx = /%/g;
+const backslashRegEx = /\\/g;
+const newlineRegEx = /\n/g;
+const carriageReturnRegEx = /\r/g;
+const tabRegEx = /\t/g;
+
+// The following function is copied from Node.js implementation of url.pathToFileURL
+// https://github.com/nodejs/node/blob/7237eaa3353aacf284289c8b59b0a5e0fa5744bb/lib/internal/url.js#L1345-L1383
+// url.pathToFileURL was added in 10.12 but pkg uses 10.4.1, and does not include this function.
+// Remove this function when pkg uses 10 LTS and includes this function.
```

Comment:
Maybe the comment should be less about PKG and more about removing this once gltf-pipeline sets Node 10 to its minimum version.

Reply:
Copied from Swayze. Yep. Good catch.